Kyuris.d.ts

```diff
@@ -9,35 +9,55 @@ export class Client extends Eris.Client {
 
     constructor(token: string, options?: KyurisClientOptions);
 
-    commands: Eris.Collection;
-    events: Set;
-    kyurisOptions: KyurisClientOptions;
-    token: string;
+    public commandFiles: Array<Command>;
+    public commands: Eris.Collection<Command>;
+    public eventFiles: Array<Event>;
+    public events: Set<Event>;
+    public kyurisHandler: KyurisHandler;
+    public kyurisOptions: KyurisClientOptions;
+    public token: string;
+
+    public run(): Promise<void>;
 
 }
 
 export class Command {
 
-    aliases: Array<string>;
-    allowInDMs?: boolean;
-    cooldown?: number;
-    meta?: object;
-    name: string;
-    nsfw?: boolean;
-    owner?: boolean;
-    permissions?: Array<string>;
-    subcommands?: Array<string>;
-    userPermissions?: Array<string>;
+    constructor();
+
+    public aliases?: Array<string>;
+    public allowInDMs?: boolean;
+    public cooldown?: number;
+    public meta?: MetaOptions;
+    public name: string;
+    public nsfwOnly?: boolean;
+    public ownerOnly?: boolean;
+    public permissions?: Array<string>;
+    public subcommands?: Array<string>;
```

Comment:
```suggestion
    public subcommands?: Array<string> | null;
```
Referring to [this line](https://github.com/NotMarx/kyuris/blob/58a970cc7f2f779ac5370244c04a9069aa0ed828/src/structures/KyurisCommand.js#L26).

Reply:
Must've missed that part.

Reply:
Fixed ([`f250886`](https://github.com/NotMarx/kyuris/pull/1/commits/f2508864c1044e0d82fdbfe2561f36a98e7d6e9f))